site/zenodo_rdm/curation/rules.py

```diff
@@ -56,3 +59,56 @@ def test_phrases_in_record(record):
         if word.lower() in record_data.lower():
             return True
     return False
+
+
+def published_before_award_start(record):
+    """Check if published before award start date."""
+    award_service = current_service_registry.get("awards")
+
+    for f in record.metadata["funding"]:
+        if f["funder"]["id"] == "00k4n6c32":
+            if award_id := f.get("award", {}).get("id"):
+                award = award_service.record_cls.pid.resolve(award_id)
+                if award.get("start_date") and (
+                    record.created < datetime.fromisoformat(award.get("start_date"))
```

Comment:
nit: we can also use `arrow.get(...)` since it's a bit shorter and consistent across the codebase